node/cn/backend.go

```diff
@@ -487,13 +487,26 @@ func (s *CN) APIs() []rpc.API {
 	governanceKlayAPI := governance.NewGovernanceKlayAPI(s.governance, s.blockchain)
 	publicGovernanceAPI := governance.NewGovernanceAPI(s.governance)
 	publicDownloaderAPI := downloader.NewPublicDownloaderAPI(s.protocolManager.Downloader(), s.eventMux)
-	tracerAPI := tracers.NewAPI(s.APIBackend)
-	unsafeTracerAPI := tracers.NewUnsafeAPI(s.APIBackend)
 
 	ethAPI.SetPublicFilterAPI(publicFilterAPI)
 	ethAPI.SetGovernanceKlayAPI(governanceKlayAPI)
 	ethAPI.SetPublicGovernanceAPI(publicGovernanceAPI)
 
+	var tracerAPI *tracers.API
+	if s.config.DisableUnsafeDebug {
+		tracerAPI = tracers.NewAPI(s.APIBackend)
+	} else {
+		tracerAPI = tracers.NewUnsafeAPI(s.APIBackend)
+		apis = append(apis, []rpc.API{
+			{
+				Namespace: "debug",
+				Version:   "1.0",
+				Service:   NewPrivateDebugAPI(s.chainConfig, s),
+				Public:    false,
+			},
+		}...)
+	}
```

Comment:
It would be more clear if you change method names like this:
 - `NewAPI` -> `NewAPIUnsafeDisabled`
 - `NewUnsafeAPI` -> `NewAPI`

Reply:
Resolved [8026919](https://github.com/klaytn/klaytn/pull/1746/commits/80269194f01f7e8fff13cf1ee335e4ca0876d5da)